### PR TITLE
[Batch] Upgrade api-version to v2021-06-01

### PR DESCRIPTION
**Pull request by cRui861**
**Description**

Upgrade the Batch data plane to 11.0.0 and management plane to 16.0.0 (API version 2021-06-01).

**Testing Guide**

Testing the Batch module is described in the [README.md](https://github.com/Azure/azurecli/blob/dev/src/azurecli/azure/cli/command_modules/batch/tests/latest/README.md)

**History Notes**

[Batch] Upgrade batch data-plane to [azure-batch 11.0.0](https://pypi.org/project/azure-batch/)
[Batch] Upgrade batch management-plane to [azure-batch-mgmt 16.0.0](https://pypi.org/project/azure-mgmt-batch/16.0.0/)
[Batch] `az batch location`: Add `list-skus` command to list SKUs available in a location
[Batch] `az batch account`: Add `outbound-endpoints` command to list outbound network dependencies

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Batch

**Comment by cRui861** [comment]
Hi @zhoxing-ms, we're looking to get these changes in before the next release. It would be great if this PR could get reviewed and approved before the cut-off for the next release happens (I believe it is this Thursday?). Thanks!

**Comment by zhoxing-ms** [comment]
@cRui861 Hi, in fact, this sprint is over, and the next sprint will be released on 09-07

File: src/azure-cli/azure/cli/command_modules/batch/_help.py [hunk 1]
```diff
@@ -162,6 +168,17 @@
 short-summary: Update the properties of a Batch job. Unspecified properties which can be updated are reset to their defaults.
 """
 
+helps['batch job stop'] = """
+type: command
+short-summary: Stop a running Batch job.
+long-summary: Terminates the specified job, marking it as completed. When a Terminate Job request is received, the Batch service sets the job to the terminating state. The Batch service then terminates any running tasks associated with the job and runs any required job release tasks. Then the job moves into the completed state. If there are any tasks in the job in the active state, they will remain in the active state. Once a job is terminated, new tasks cannot be added and any remaining active tasks will not be scheduled.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/batch/_help.py:174]: Please use the first person voice
```suggestion
long-summary: Terminate the specified job, marking it as completed. When a Terminate Job request is received, the Batch service sets the job to the terminating state. The Batch service then terminates any running tasks associated with the job and runs any required job release tasks. Then the job moves into the completed state. If there are any tasks in the job in the active state, they will remain in the active state. Once a job is terminated, new tasks cannot be added and any remaining active tasks will not be scheduled.
```

**Review by zhoxing-ms — CHANGES_REQUESTED** [review]
Could you please add some test cases for those new commands?

File: src/azure-cli/azure/cli/command_modules/batch/_help.py (hunk 1 above)
Comment (gingi) [reply src/azure-cli/azure/cli/command_modules/batch/_help.py:174]: Fixed.

**Comment by gingi** [comment]
> Could you please add some test cases for those new commands?

I've added a few test cases for the new commands.

**Review by zhoxing-ms — APPROVED** [review]

**Review by zhoxing-ms — CHANGES_REQUESTED** [review]
@gingi If you want to release it in the next week, please submit this PR to the release branch, because we will launch the hot fix process for it.

**Comment by cRui861** [comment]
@zhoxing-ms Updated the branch to be merged into the `release` branch instead of `dev`.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**